src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/Signatory.tsx

```diff
@@ -151,32 +149,27 @@ export const Signatory = ({
   }, [query, isOwnAccount, contacts, contactsFiltered]);
 
   const onNameChange = (newName: string) => {
-    setName(newName);
-    signatoryModel.events.signatoriesChanged({
-      index: signtoryIndex,
+    signatoryModel.events.changeSignatory({
+      index: signatoryIndex,
       name: newName,
-      address,
+      address: signatoryAddress,
     });
   };
 
   useEffect(() => {
-    if (displayName !== name) {
+    if (displayName && displayName !== signatoryName) {
       onNameChange(displayName);
     }
   }, [displayName]);
 
   const onAddressChange = (newAddress: string) => {
-    if (!validateAddress(newAddress)) {
-      setAddress('');
+    const validatedAddress = validateAddress(newAddress) ? newAddress : '';
+    const fixedAddress = toAddress(validatedAddress, { prefix: chain.value.addressPrefix });
```

Comment:
I am not correctly understand, what is the `fixedAddress` means? Address with prefix?

Reply:
In prev implementation address was saved as is, but in combobox it got formatted with chain prefix.